src/patternfly/components/Select/examples/Select.md

```diff
@@ -156,10 +150,6 @@ The single select typeahead should be used when the user is selecting one option
 ```
 
 The Dropdown Multi Select should be used when the user is selecting multiple items from a list. The user can narrow the list by typing from the keyboard. The List updates while typing. Selected items create a new chip and are removed from the list. The user may clear selections individually or all at once to restore the placeholder.
-### Accessibility
-| Attribute | Applied to | Outcome |
-| -- | -- | -- |
-| `aria-selected="true"` | `.pf-c-select__menu-item` | Should be set programmatically to indicate the active item. |
```

Comment:
are these no longer valid? we still set `aria-selected="true"` on  select menu items

Reply:
Axe was complaining that `aria-selected` is not valid on button elements. I'll double check that it was always button elements...